### PR TITLE
TrOCR model implementation on LPR

### PR DESCRIPTION
**Pull request by BADRI-1999**
trocr printed images based model, checked on test images and it's working decent. Review the code once.

### CODE REVIEW

**Review by arponitis — APPROVED** [review]

**Review by RudrakshSugandhi — COMMENTED** [review]
How much accuracy the TROcr Provide  would like to compare it with easyOCR.

**Review by VarshithaY — APPROVED** [review]

**Comment by BADRI-1999** [comment]
> How much accuracy the TROcr Provide would like to compare it with easyOCR.

We are not training our model from scratch, we are pretrained model only

**Review by RudrakshSugandhi — APPROVED** [review]
I was just wondering about the accuracy of the pre-trained model but we can verify it later. 
Approving the pull request
